app/views/layouts/application.html.erb

```diff
@@ -19,7 +19,7 @@
   <meta property="og:url"                content="https://gambe.ro"/>
 <meta property="og:type"               content="website" />
 <meta property="og:title"              content="Gambe.ro" />
-<meta property="og:description"        content="Gambe.ro: la comunità italiana dei programmatori" />
+<meta property="og:description"        content="Gambe.ro: la comunità per discutere, capire, creare tecnologia" />
```

Comment:
..la comunità _italiana_ per discutere..

Reply:
abbiamo omesso italiana volontariamente perché: beh è in Italiano, ci risparmiamo una parola che già è una desc bella lunga

Reply:
Mmhm. Una idea che non mi era venuta prima: se omettessimo _comunità_ del tutto? `Gambe.ro: discutere, capire, creare tecnologia`.

Reply:
era stato considerato nella discussione su proposta mia, ma la versione attuale convinceva più persone.